sycl/source/detail/devicelib/fallback-cassert.cpp

```diff
@@ -8,9 +8,7 @@
 
 #include "wrapper.h"
 
-// __attribute((format(...))) enables compiler checks for a format string.
-int __spirv_ocl_printf(const __attribute__((opencl_constant)) char *fmt, ...)
-    __attribute__((format(printf, 1, 2)));
+#include <CL/__spirv/spirv_ops.hpp>
```

Comment:
Do we really need all the spirv_ops included into device library?
@asavonic , don't you have any objections?

Reply:
The file should contain only declarations, right? I don't see a problem.

Reply:
It contains some definitions, but only for host, I suppose. https://github.com/intel/llvm/blob/54dddb4b0e7740836c90048a28311fd579227814/sycl/include/CL/__spirv/spirv_ops.hpp#L237